apis/v1alpha1/instrumentation_types.go

```diff
@@ -22,6 +22,10 @@ import (
 
 // InstrumentationSpec defines the desired state of OpenTelemetry SDK and instrumentation.
 type InstrumentationSpec struct {
+	// Selector is the selector label of injected Object
+	// +optional
+	Selector map[string]string `json:"selector,omitempty"`
```

Comment:
Can we use a https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#LabelSelector for this instead? This is the effective K8s standard for selecting resources based on labels.

Reply:
Done, PTAL.